proxies/go/handler_client.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/devcyclehq/go-server-sdk"
+	"net/http"
+)
+
+type clientRequestBody struct {
+    ClientId         string   `json:"clientId"`
+    SdkKey      string   `json:"sdkKey, omitempty"`
+    EnableCloudBucketing bool `json:"enableCloudBucketing"`
+    WaitForInitialization bool `json:"waitForInitialization"`
+    Options devcycle.DVCOptions `json:"options"`
+}
+
+type clientResponseBody struct {
+    Message         string   `json:"message, omitempty"`
+    AsyncError      string   `json:"asyncError, omitempty"`
+    Exception       string   `json:"exception, omitempty"`
+}
+func clientHandler(w http.ResponseWriter, r *http.Request) {
+    var reqBody clientRequestBody
+    err := json.NewDecoder(r.Body).Decode(&reqBody)
+    if err != nil {
+        http.Error(w, err.Error(), http.StatusBadRequest)
+        return
+    } else if reqBody.ClientId == "" {
+        http.Error(w, "Invalid request: missing clientId", http.StatusBadRequest)
+        return
+    }
+
+	reqBody.Options.DisableLocalBucketing = reqBody.EnableCloudBucketing
+	client, err := devcycle.NewDVCClient(reqBody.SdkKey, &reqBody.Options, nil)
+	var res clientResponseBody
+	if err != nil {
```

Comment:
don't know if github is being weird but there's an indentation here

Reply:
yea it looks normal on my end so im not sure lol